App.tsx

```diff
@@ -114,43 +112,44 @@ export default function App() {
   const [isSetUp, setIsSetUp] = useState(true);
   const [hasAccepted, setHasAccepted] = useState(false);
 
-  useEffect(async () => {
-    try {
-      await AsyncStorage.getItem("@isSetUp2").then((response) => {
-        if (response == null) {
-          setIsSetUp(false);
-        }
-      });
-    } catch (e) {
-      console.log(e);
-    }
+  useEffect(() => {
+    (async () => {
+      try {
+        await AsyncStorage.getItem("@isSetUp2").then((response) => {
+          if (response == null) {
+            setIsSetUp(false);
+          }
+        });
+      } catch (e) {
+        console.log(e);
+      }
```

Comment:
it's better to use `console.error` here imo

```suggestion
      } catch (err) {
        console.error(err);
      }
```

Reply:
Yes indeed, but I have not already refactored this

Reply:
*yet 😛